lockbox-ios/FxA/FxAWebView.swift

```diff
@@ -0,0 +1,25 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import WebKit
+import Foundation
+
+class FxAWebView : WKWebView, WKNavigationDelegate {
+    let oauthBaseURL = "https://scoped-keys-oct10.dev.lcip.org/"
```

Comment:
why the separate values here?

Reply:
The base URL will presumably change once these are in prod. FxAWebView is only sketched out right now, also -- I can remove it from this commit.

Reply:
Ideally, we use [configuration discovery](https://developer.mozilla.org/en-US/docs/Mozilla/Tech/Firefox_Accounts/Introduction#Endpoint_Discovery) to configure the endpoints.  I'd like to do that from the WebExtension, but caching needs to be figured out.  In the meantime, its static configuration is essentially a superset of the configuration info.

That all said, it's not a blocker for me either way.